RunoPW/dist/metadata.json

```diff
@@ -0,0 +1,22 @@
+{
+  "author": {
+    "name": "め Kerem",
+    "id": "665482610859966464"
+  },
+  "contributors": [{
+    "name": "Runo",
+    "id": "702935358395908168"
+  }],
+  "service": "Runo",
+  "description": {
+    "en": "Make Friends & Chat With Millions In A Virtual World",
+    "tr": "Müthiş İnsanlarla Dolu İlginç Bir Yer"
+  },
+  "url": "runo.pw",
+  "version": "1.0.0",
+  "logo": "https://imgur.com/utrvCGI",
+  "thumbnail": "https://imgur.com/utrvCGI",
+  "color": "#ffff00",
+  "tags": ["Runo", "Sanal Dünya"],

```

Comment:
Tags should be used as alternate names whenever possible, shortened versions must be included as well (e.g. if an Amazon presence had included AWS support it would have its tags like : "amazon-web-services" and "aws"). This is not required if it's not possible, but will make it easier for users when searching.
Tags must not include any spaces, slashes, single/double quotation marks, unicode characters and should always be lowercase.